### PR TITLE
[DPE-2670] Update folder structure to reflect new snap

### PR DESCRIPTION
**Pull request by phvalguima**
Following snap change: https://github.com/canonical/opensearch-snap/commit/f8353c5caa90a93666a613e7eb64b728eaa1f30a

Update the charm to track this new folder structure.

### CODE REVIEW

File: src/opensearch.py [hunk 1]
```diff
@@ -104,6 +104,21 @@ def is_failed(self) -> bool:
 
         return service_failed("snap.opensearch.daemon.service")
 
+    def _set_env_variables(self):
```
Comment (Mehdi-Bendriss) [inline src/opensearch.py:107]: Why are these defined? these should be already defined from within the snap

Comment (phvalguima) [reply src/opensearch.py:107]: And add a `@override`

**Review by Mehdi-Bendriss — APPROVED** [review]
Thank you! We'll revisit the list of used env variables on [DPE-2679](https://warthogs.atlassian.net/browse/DPE-2679)

**Comment by phvalguima** [comment]
@Mehdi-Bendriss I manually ran the tests:
* ha-base-integration: https://pastebin.canonical.com/p/3hNtH2X6X3/
* ha-networking-integration: https://pastebin.canonical.com/p/VJkBjPJ9x6/

Both succeeded.

**Review by delgod — APPROVED** [review]

**Pull request merged**